textprocessors.h

```diff
@@ -13,7 +14,7 @@ void lowercase_letters(char *text);
 
 // Johnny Text Processors
 void leet_case(char *text);
-void capitalize_letters(char *text);
+void capitalize_letters(char * const text);
```

Comment:
```suggestion
void capitalize_letters(char const * text);
```
Update header signature to match `textprocessors.cpp`

Reply:
```suggestion
void capitalize_letters(char * text);
```
Update header signature to match `textprocessors.cpp`